examples/experiment_eval.py

```diff
@@ -147,7 +148,7 @@ def _iterate_env_agent_combo_not_in_db(
                 terminal_evaluators=[
                     ReachGoalLLMEvaluator(
                         model_names["env"],
-                        EvaluationForTwoAgents[SotopiaDimensions],
+                        EvaluationForTwoAgents[EvaluationByGroup],
```

Comment:
Please don't just directly replace the `SotopiaDimensions`, make this a hyperparameter? And the default is to use `SotopiaDimensions`

Reply:
Agree. That needs to be specified in config file